src/stores/root.ts

```diff
@@ -98,13 +98,18 @@ export class RootStore {
 					},
 				},
 				chainOpts: this.chainStore.chainInfos.map(chainInfo => {
+					let gas = 500_000;
+					if (chainInfo.chainId.startsWith('osmosis-')) {
+						gas = 1_350_000;
+					} else if (chainInfo.chainId.startsWith('secret-')) {
+						gas = 25_000;
+					} else if (chainInfo.chainId.startsWith('columbus-') || chainInfo.chainId.startsWith('cosmoshub-')) {
+						gas = 100_000;
+					}
```

Comment:
Can we revert this clause? Then looks good to merge to me

Reply:
Yes, done. :+1: